circuits/cpp/src/aztec3/circuits/kernel/public/.test.cpp

```diff
@@ -361,12 +361,12 @@ PublicKernelInputs<NT> get_kernel_inputs_with_previous_kernel(NT::boolean privat
         .bytecode_hash = 1234567,
     };
 
-    CombinedHistoricTreeRoots<NT> const historic_tree_roots = { .private_historic_tree_roots = {
-                                                                    .private_data_tree_root = 1000,
-                                                                    .contract_tree_root = 2000,
-                                                                    .l1_to_l2_messages_tree_root = 3000,
-                                                                    .private_kernel_vk_tree_root = 4000,
-                                                                } };
+    // CombinedHistoricTreeRoots<NT> const historic_tree_roots = { .private_historic_tree_roots = {
```

Comment:
Should we remove instead?

Reply:
Probably? I haven't looked into the history or discussed with core circuits team yet.

Reply:
Yeah I didn't see it was a draft, my bad

Reply:
I ended up making an issue to look at this and two other unused variables in the same test file, and I reference that issue in TODOs in the code.